PocketKit/Sources/PocketKit/Settings/DeleteAccountView.swift

```diff
@@ -0,0 +1,143 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Textile
+
+@MainActor
+struct DeleteAccountView: View {
+    /// If the user is premium
+    @State var isPremium: Bool
+
+    /// Confirmation by the user that they have cancelled their premium account
+    @State var hasCancelledPremium: Bool = false
+
+    /// Confirmation by the user they they understand the deletion is permanent
+    @State var understandsPermanentDeletion: Bool = false
+
+    /// State variable listened on from our view model
+    @State var isPresentingCancelationHelp: Bool
+
+    var deleteAccount: () -> Void
+
+    @Environment(\.dismiss)
+    var dismiss
+
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 0) {
+                Text(L10n.Settings.AccountManagement.deleteYourAccount)
+                    .style(.deleteAccountView.header)
+                    .padding(.top, 50)
```

Comment:
not sure if we can across a consensus, but we discussed about possible putting magic numbers in an `enum`
https://pocket.slack.com/archives/C01UH57EJKD/p1677173129600309

Reply:
Making a note to come back to this.